### DISCUSSION TITLE
Deleting of Flow in Authentication

### DISCUSSION THREAD

**Discussion by akshayInCode** [keycloak/keycloak#32691] [Q&A] [answered]
I have created a flow under authentication, which I need to delete because  it s not working for process I want. I have tried but not getting option to delete as it is also showing used by browser flow. please help want to delete custom authentication flow.
![Screenshot (317)](https://github.com/user-attachments/assets/1b9464ee-c219-4576-b963-80677cff3b6e)

**Comment by darius-m** [comment, marked as answer]
Bind the `browser` built-in flow for the `Browser flow` first, and then you should be able to delete the custom one.

**Comment by akshayInCode** [comment]
Thanks , done.
